srcpkgs/xinit/patches/06_move_serverauthfile_into_tmp.diff

```diff
@@ -0,0 +1,11 @@
+--- startx.cpp.orig
```

Comment:
Can you add the header of the patch too? Otherwise it's hard to see why this patch was added.